### ISSUE TITLE
Can the order of toolbars be changed by the user?

### ISSUE THREAD

**Issue by kleinbottle** [texstudio-org/texstudio#2062]
### Describe the feature and the current behavior/state
The `File` toolbar with the `Save` icon is at present shown leftmost. Being right-handed, I would prefer to move it to the right. I think that is not possible at present. Hence my request.
### Who will benefit with this feature?
Right-handed folk plus those who are used to the `Save` icon being on the right.
### Any Other info
Being able to shuffle the order of the toolbars will notch up the comfort of customizability.\ even higher.

**Comment by sunderme** [comment]
you can move the toolbars around.

**Comment by kleinbottle** [comment]
> you can move the toolbars around.

I have just started using `TeXStudio`. So, I am not familiar with how this is done. Is there a video link to do that?

**Comment by sunderme** [comment]
just click and hold on the "handlers" at the left hand side of the toolbars.

**Comment by kleinbottle** [comment]
> just click and hold on the "handlers" at the left hand side of the toolbars.

It took a little getting used to but I was able to reposition the icons. I now have two toolbar panels. Is there any way of abbreviating some less used ones or do I configure that one by one in the `Toolbar` configuration panel?

**Comment by sunderme** [comment]
in options/toolbars you can adapt each toolbar individually.

**Comment by kleinbottle** [comment]
> in options/toolbars you can adapt each toolbar individually.

Yes.I experimented for some time and got the hang of it. I must say that some themes and colours did not displayp the "handlers" with enough contrast. But now I can see them all and it is not unlike KDE `kate` which I am familiar with. I used to use `kile` but it is not developed anymore. I am glad that `TeXStudio` does what `kile` did and much much more. Thanks.
